currency_rate_update_TH_BOT/models/res_currency_rate_provider_BOT.py

```diff
@@ -87,6 +87,12 @@ def _get_supported_currencies(self):
             "PKR",
         ]
 
+    def _get_currency_unit(self, bot_currency_name):
```

Comment:
```suggestion
    def _get_currency_unit(self, bot_currency_name):
```
please add description for this method.